server/planning/events/events_post.py

```diff
@@ -90,20 +92,20 @@ def validate_post_state(new_post_state):
         except AssertionError:
             abort(409)
 
-    @staticmethod
-    def validate_item(doc):
-        errors = get_resource_service("planning_validator").post(
-            [{"validate_on_post": True, "type": "event", "validate": doc}]
-        )[0]
-
-        if errors:
-            # We use abort here instead of raising SuperdeskApiError.badRequestError
-            # as eve handles error responses differently between POST and PATCH methods
-            abort(400, description=errors)
+    # TODO-ASYNC: Uncomment method when service is changed to async to allow new async validate_docs function
+    # @staticmethod
+    # def validate_item(doc):
+    #     errors_list = await validate_docs([{"validate_on_post": True, "type": "event", "validate": doc}])
+    #     errors = errors_list[0]
+    #
+    #     if errors:
+    #         # We use abort here instead of raising SuperdeskApiError.badRequestError
+    #         # as eve handles error responses differently between POST and PATCH methods
+    #         abort(400, description=errors)
```

Comment:
@MarkLark86 for cases like this where we now have to use the new functions and the old service has been deleted, I have commented the functions out and where they are used until the service is converted to async. 

Thoughts ?

Reply:
No worries, the to-do comments should help us search for outstanding work